07-routing-gossip.md

```diff
@@ -285,6 +285,7 @@ The following `address descriptor` types are defined:
          onion service addresses; Encodes:
          `[32:32_byte_ed25519_pubkey] || [2:checksum] || [1:version]`, where
          `checksum = sha3(".onion checksum" | pubkey || version)[:2]`.
+   * `5`: DNS hostname; data = `[1:hostname_len][hostname_len:hostname][2:port]` (length up to 258)
```

Comment:
This should specify either that it needs to be ASCII (punycode for IDNs) or UTF-8. Meeting discussion was a bit of a toss-up, but it seems at the end we had some agreement that the field should be required to be ASCII (with a link to the wikipedia for punycode).